primitives/hp-proof-of-existence/src/lib.rs

```diff
@@ -94,3 +95,25 @@ impl sp_inherents::InherentDataProvider for InherentDataProvider {
 pub trait OnProofVerified {
     fn on_proof_verified(pubs_hash: H256);
 }
+
+/// Wrapper around u32 for MaxStorageAttestations
```

Comment:
Given that `u64` was used for attestation ids in the pallet, would it make sense to make also this a `u64` to have things more consistent?